src/app/home/home.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { BorrowerService } from '../service/borrowerService/borrower.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
+
+  borrower: any = null;
+
+  constructor(private borrowerService: BorrowerService) { }
+
+  ngOnInit() {
```

Comment:
i would add a router here that checks if Borrower is logged in(I added that to the borrower service in my recent commit) and then it reroutes them to the login page if not logged in